package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@learnersguild/game-cli",
-  "version": "0.7.3",
+  "version": "0.7.4",
```

Comment:
technically this should be a major version change since it's backwards incompatible.